### ISSUE TITLE
Help panel font size doesn't apply to all text in help

### ISSUE THREAD

**Issue by dracodoc** [rstudio/rstudio#6425]
#6333 # System details

    RStudio Edition : Desktop 1.3.842
    RStudio Version : 
    OS Version      :  windows
    R Version       :  3.6.2

### problem

Previously I need to edit `RStudio\resources\R.css` body font size to increase the help text size.

```
body, td {
   font-family: sans-serif;
   font-size: 13pt;
}
```

With newer version RStudio we have an option to set help panel text in option-advanced. However this doesn't apply to some part in help txt, lots of text are very small no matter what font size is selected. The option seemed to only apply to part of them.

![image](https://user-images.githubusercontent.com/1730496/76145871-d7fa0a80-605b-11ea-8b3a-a52e7ff0539f.png)

**Comment by melissa-barca** [comment]
Thank you for bringing this to our attention! I've submit PR #6426 to fix the issue.

**Comment by adamconroy** [comment]
Verified fixed in v1.3.929-1
